sklearn/model_selection/_split.py

```diff
@@ -2180,3 +2181,54 @@ def _build_repr(self):
         params[key] = value
 
     return '%s(%s)' % (class_name, _pprint(params, offset=len(class_name)))
+
+
+class GroupTimeSeriesSplit(TimeSeriesSplit):
```

Comment:
Please add a docstring describing expected functionality.

Reply:
@jnothman - Added  the docstring . Can you please check .